### PR TITLE
Fixed page not autoscrolling to top bug

### LINKED ISSUE #3015: Page scroll remains fixed when navigating
**Feature to document**
When navigating, if I've scrolled to the bottom of one page and navigate to another page, then the new page loads with scroll fixed at the bottom, One has to scroll up to start reading from starting of the page.
To reproduce, scroll down to the bottom in any page, and then click on any link to load another page.

**Author(s)**
N/A

**Additional information**
window.scrollTo(0,0) might save the problem. I'll send a PR.

### PR DESCRIPTION
**Pull request by nveenjain**
Fixes #3015  
Added window.scrollTo(0,0) on page mounting.

### CODE REVIEW

File: src/components/Page/Page.jsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ class Page extends React.Component {
   }
 
   componentDidMount() {
+    window.scrollTo(0,0);
```
Comment (EugeneHlushko) [inline src/components/Page/Page.jsx]: What about anchor links?

e.g. `page#section` ?

Comment (montogeek) [reply src/components/Page/Page.jsx]: What about if you reopen the last closed tab?

Comment (EugeneHlushko) [reply src/components/Page/Page.jsx]: Yeah this approach isn't good enough. What if we bind onto menu link?

Comment (nveenjain) [reply src/components/Page/Page.jsx]: Anchor links work fine, for last closed tab, I'm thinking of storing scroll positions when navigating the code and restoring it back to the stored position. This way if a user has read upto a certain position, his/her position will be preserved when he/she comes back to the same page.

**Comment by montogeek** [comment]
This definitely breaks the page when:
1. Open https://deploy-preview-3016--webpackjsorg-netlify.netlify.com/contribute/writing-a-loader/#testing
2. Close the tab.
3. Re open it using reopen last tab option of your browser, page stays at the top.
4. It should open in the same anchor when it was closed.

This behaviour works fine in webpack.js.org.

**Comment by montogeek** [comment]
@nveenjain Friendly ping

**Comment by montogeek** [comment]
Closing because it got stalled.

**Pull request closed without merging**